### DISCUSSION TITLE
Localization - Page layout

### DISCUSSION THREAD

**Discussion by MrNikolovski** [GrapesJS/grapesjs#6181] [Q&A]
Hi all. 

Apologies if this question has been asked and answered previously but is there a way to create custom blocks and localize their names so they show the correct name when the language is changed? Both on the canvas and in the page layout manager. 

Tried to look at the translated files under locale but without luck. 

Thank you.

**Comment by artf** [comment]
[Here](https://github.com/GrapesJS/grapesjs/blob/dev/packages/core/src/i18n/locale/en.js) you can find the references for i18n keys for each module.
